utils/setup_cp_demo.sh

```diff
@@ -1,6 +1,13 @@
 #!/bin/bash
 
-DEFAULT_CP_DEMO_HOME=$(realpath ${MONITORING_STACK}/../../cp-demo)
+fullpath ()
```

Comment:
For your consideration: add a comment that explains why this workaround instead of just including `realpath`

Reply:
Thanks for the feedback, I've added a comment.